templates/search-bar.mustache

```diff
@@ -2,6 +2,14 @@
 	<div class="search-wrap" style="display:none;">
 		<span class="search-icon"></span>
 		<span class="search-icon-cancel"></span>
-		<input type="text" class="search" placeholder="Search - [ctrl + k]" autocomplete="off" autocorrect="off" autocapitalize="none" spellcheck="false" />
+		<input
+			type="text"
+			class="search"
+			placeholder="Search - [ctrl + k]"
+			autocomplete="off"
+			autocorrect="off"
+			autocapitalize="none"
+			spellcheck="false"
+			tabindex="1"/>
```

Comment:
After reading [some info on tabindex](http://adrianroselli.com/2014/11/dont-use-tabindex-greater-than-0.html), I don’t think we should set it to 1. I think it’s fine to leave it in the default order.